### DISCUSSION TITLE
Wonderful tool!

### DISCUSSION THREAD

**Discussion by mercster** [nefares/Background-Muter#23] [General]
I don't understand why more people don't need / want / think about this feature.  We have modern multitasking operating systems with multiple monitors; for some reason, people are content to have applications blaring in the background.  I have used Borderless Gaming tool in the past for this, but it interferes with the window manager on many applications.  Your tool is the chef's kiss for multitaskers!

If you put up a donate button I will throw you a couple of dollars!  Merry Christmas.

**Comment by nefares** [comment]
@mercster I have also used BG in the past and it has a lot of features, which may be interesting for some people but not for those just wanting the "simple" task of muting background apps.

 Anyway thank you for the kind words! I'm very happy to hear that the tool is helpful to you and the community!

**Comment by Chargin** [comment]
I cannot see an exe file in the root or any other folders?

**Reply by mercster** [reply]
Sounds like you downloaded a package with the source, instead of a release package with the compiled binary.

**Reply by Chargin** [reply]
Aha ok looking for the correct file now.

**Comment by olegx-smirnov** [comment]
Big thanks, friend!
